ajax/libs/jsrsasign/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jsrsasign",
-  "version": "7.1.5",
+  "version": "7.1.3",
```

Comment:
This should be modified in the commit "Modify jsrsasign from npm to use git auto-update"?